portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +14,28 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    ArrayList<String> Greetings = new ArrayList();
+    ArrayList<String> commentRepo = new ArrayList<String>();
```

Comment:
have these as private final List<String> for both. Also naming convention should be greetingsList and commentsList